tests/test_events.py

```diff
@@ -564,3 +564,40 @@ def test_meeting_stream_link_unavailable(mocker):
 
     assert isinstance(current_meeting, QuerySet)
     assert len(current_meeting) == 0
+
+
+@pytest.mark.parametrize('event_name,expected_live_comment_value', [
+    ('Special Operations, Safety, and Customer Experience Committee', True),
+    ('Planning and Programming Committee', True),
+    ('Operations, Safety, and Customer Experience Committee', True),
+    ('Finance, Budget and Audit Committee', True),
+    ('Executive Management Committee', True),
+    ('Construction Committee', True),
+    ('Measure R Independent Taxpayer Oversight Committee', False),
+    ('Measure M Independent Taxpayer Oversight Committee', False),
+    ('Independent Citizen’s Advisory and Oversight Committee', False),
+])
+def test_live_comment_info_displays_appropriately(client,
+                                                  event,
+                                                  event_document,
+                                                  event_name,
+                                                  expected_live_comment_value):
+
+    in_an_hour = LAMetroEvent._time_from_now(hours=1).strftime('%Y-%m-%d %H:%M')
+    test_event = event.build(name=event_name, start_date=in_an_hour)
+    event_document.build(note='Agenda', event_id=test_event.id)
+
+    assert test_event.accepts_live_comment == expected_live_comment_value
+
+    url = reverse('lametro:events', args=[test_event.slug])
+    response = client.get(url)
+    response_content = response.content.decode('utf-8')
+
+    live_comment_signature = (
```

Comment:
you are testing both a model and a template rending here. please split into two tests. the test for the template just needs a true and false condition.

Reply:
Great call, done in https://github.com/datamade/la-metro-councilmatic/pull/823/commits/8c8383b851a5df352d9c5d361edcc8a5168c9979.